pages/agenda.tsx

```diff
@@ -51,6 +52,9 @@ export default function Agenda() {
           <div className='agenda-nav'>
             {NavBar(1)}
             {NavBar(2)}
+            <div className="compact button">
+              <Link href={'/agenda-compact'}>Compact View</Link>
+            </div>
```

Comment:
Let's add the button as the first element of the agenda nav with a normal size button.